orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,69 @@
+from django.db import migrations, transaction
+from django.utils import timezone
+import datetime
+
+DOG_OWNER_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23',
+                '4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']
+
+DAYCARE_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
+              '22', '23', '23', '22', '21', '20', '19', '18', '17', '16', '15', '14', '13', '12', '11', '10', '9', '8',
+              '7', '6', '5', '4']
+
+START_DATE = [timezone.datetime(2022, 3, 23), timezone.datetime(2022, 5, 23), timezone.datetime(2022, 2, 23),
+              timezone.datetime(2022, 6, 7), timezone.datetime(2022, 7, 15), timezone.datetime(2021, 1, 20),
+              timezone.datetime(2023, 6, 6), timezone.datetime(2021, 5, 6), timezone.datetime(2023, 4, 20),
+              timezone.datetime(2022, 3, 30), timezone.datetime(2022, 7, 25), timezone.datetime(2022, 10, 10),
+              timezone.datetime(2022, 12, 15), timezone.datetime(2022, 5, 11), timezone.datetime(2022, 9, 10),
+              timezone.datetime(2022, 6, 11), timezone.datetime(2022, 5, 5), timezone.datetime(2022, 2, 22),
+              timezone.datetime(2022, 7, 8), timezone.datetime(2022, 8, 1), timezone.datetime(2022, 1, 1),
+              timezone.datetime(2022, 7, 7), timezone.datetime(2022, 7, 20), timezone.datetime(2022, 8, 27),
+              timezone.datetime(2022, 1, 15), timezone.datetime(2022, 12, 18), timezone.datetime(2022, 9, 23),
+              timezone.datetime(2022, 11, 12), timezone.datetime(2022, 11, 11), timezone.datetime(2022, 1, 23),
+              timezone.datetime(2022, 6, 12), timezone.datetime(2022, 2, 15), timezone.datetime(2022, 3, 11),
+              timezone.datetime(2022, 1, 17), timezone.datetime(2022, 12, 23), timezone.datetime(2022, 3, 12),
+              timezone.datetime(2022, 3, 5), timezone.datetime(2022, 4, 4), timezone.datetime(2022, 5, 15),
+              timezone.datetime(2022, 5, 5)]
+
+END_DATE_DELTA = [7, 2, 8, 3, 6, 2, 7, 2, 1, 5,
+                  15, 4, 11, 3, 9, 2, 5, 7, 3, 4,
+                  10, 14, 15, 3, 8, 6, 7, 6, 4, 5,
+                  9, 7, 5, 2, 3, 17, 8, 2, 1, 3]
+
+
```

Comment:
Good Job @OfirMatasas 
In general,  wherever you have created a list of values I think you should insert the values in a loop or use a list comprehension instead of the existing situation.

Reply:
Good idea.
Anyway, I've changed the code to Jenia's suggestion, so it's no longer relevant.